### DISCUSSION TITLE
Add export gpx files

### DISCUSSION THREAD

**Discussion by mrwolf78** [teslamate-org/teslamate#722] [Ideas]
I would like the possibility to export drive details to a gpx file. 

Why? Because sometimes I drive a nice route, and I'd like to share or ride it again later, which would be easier if I had a gpx file (or other shareable format)

**Comment by brendanpward** [comment]
I too would like to see this implemented as a feature.

**Comment by adriankumpf** [comment]
There is already a PR for it here: https://github.com/adriankumpf/teslamate/pull/993

**Comment by kreativmonkey** [comment]
There is actually no way to export a complete trip over multiple days? Only one drive can be exported?

**Comment by tijszwinkels** [comment]
I ended up writing a separate utility for writing gpx-files based on data from the Tesla api. It's available here: https://github.com/tijszwinkels/tesla-gps
